backend/glific_migration/sync_assistant/sync_assistant.py

```
@@ -0,0 +1,25 @@
+from pathlib import Path
+import json
+from glific_migration.sync_assistant.processor import AssistantIngestProcessor
+
+base_dir = Path(__file__).parent.resolve()
+
+
+def main():
+    with open(base_dir / "../config.json", "r") as file:
```

Ruff (0.12.2)

Unnecessary mode argument

Remove mode argument

(UP015)